### PR TITLE
add nano pytorch training examples

### PR DESCRIPTION
**Pull request by yangw1234**
## Description

Add nano pytorch training examples:

1. use ipex
2. use multi-instance training

### CODE REVIEW

File: python/nano/tutorial/training/pytorch-lightning/lightning_train_ipex.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch-lightning/lightning_train_ipex.py:1]: just `train_ipex.py`

File: python/nano/tutorial/training/pytorch-lightning/lightning_train_ipex.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    return train_dataloader, val_dataloader
+
+
+if __name__ == "__main__":
+
+    model = MyLightningModule()
+    train_loader, val_loader = create_dataloaders()
+
+    # IPEX accelerated training by setting use_ipex=True
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch-lightning/lightning_train_ipex.py:96]: Briefly example IPEX

File: python/nano/tutorial/training/pytorch-lightning/lightning_train_multi_instance.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+    return train_dataloader, val_dataloader
+
+
+if __name__ == "__main__":
+
+    model = MyLightningModule()
+    train_loader, val_loader = create_dataloaders()
+
+    # Multi-instance training by setting num_processes=2
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch-lightning/lightning_train_multi_instance.py:96]: briefly explain multi-instance training

**Comment by yangw1234** [comment]
tests will be added later

**Pull request merged**